### PR TITLE
Run integration tests in a custom namespace (and cleanup via ns deletion)

### PR DESCRIPTION
**Pull request by mwielgus**

### CODE REVIEW

**Comment by vishh** [comment]
LGTM.

**Comment by mwielgus** [comment]
In Jenkins, Heapster service has repeating problems with finding a matching pod (in the namespace) what is quite odd as it works perfectly fine on my computer/cluster.

**Comment by vishh** [comment]
That's weird. Shall we try bumping up the Kubernetes release ? It is currently set to v1.0.1.

**Comment by mwielgus** [comment]
I bumped the version to 1.0.5. Integration tests pass (travis failed because of flaky test - filled an issue for that #565). I will start yet another test run to confirm that we are good now.

File: integration/heapster_api_test.go [hunk 1]
```diff
@@ -630,10 +625,10 @@ func apiTest(kubeVersion string) error {
 	if err := createAll(fm, ns, &svc, &rc); err != nil {
 		return err
 	}
-	if err := fm.WaitUntilPodRunning(ns, rc.Spec.Template.Labels, time.Minute); err != nil {
+	if err := fm.WaitUntilPodRunning(ns, rc.Spec.Template.Labels, 5*time.Minute); err != nil {
```
Comment (mvdan) [inline integration/heapster_api_test.go]: Why are we bumping the timeout here and below?

Comment (mwielgus) [inline integration/heapster_api_test.go]: Just to check if this helps. There were multiple runs (only in Jenkins) where service could not find his endpoints within the previous timeouts.

**Comment by mwielgus** [comment]
Getting rid of system-like stuff from the configs and changing the labels a bit helped. All tests pass. Merging

File: integration/heapster_api_test.go (hunk 1 above)
Comment (mwielgus) [inline integration/heapster_api_test.go]: Reverted

**Pull request merged**